mm2src/mm2_bitcoin/spv_validation/src/work.rs

```diff
@@ -244,7 +244,13 @@ pub(crate) mod tests {
             Ok(None)
         }
 
-        async fn remove_headers_up_to_height(&self, _height: u64) -> Result<(), BlockHeaderStorageError> { Ok(()) }
+        async fn remove_headers_from_storage(
+            &self,
+            _from_height: u64,
+            _to_height: u64,
+        ) -> Result<(), BlockHeaderStorageError> {
+            Ok(())
+        }
```

Comment:
This seems ineffective function, can you `info!()` here?

Reply:
This is an implementation of the trait for unit tests related to calculating the required work/bits for the next btc block headers where `remove_headers_from_storage` is not used. I don't think there is a need to add `info!()` or anything else here for now.